### PR TITLE
switch to named bonuses

### PR DESCRIPTION
**Pull request by jeffkaufman**
Right now you can easily have merge conflicts if two PRs both try to give the same user points.

Use the directory trick to tell git these are independent: give each player a directory of named bonuses, and add them up to get points.

I've added this adding-up to #33

### CODE REVIEW

**Comment by TheJhyde** [comment]
This seems... complicated. Are merge conflicts really so bad? I'm not going to reject this but I won't approve.

**Comment by jeffkaufman** [comment]
@TheJhyde 

> This seems... complicated. Are merge conflicts really so bad? I'm not going to reject this but I won't approve.

Merge conflicts mean that every time something gets merged to master all outstanding PRs need to start over with getting approved.  So if we want points to be easy to use then I think this is a lot better.

**Comment by pavellishin** [comment]
@jeffkaufman whoops, conflicts now.

**Comment by tnelling** [comment]
If that doesn't prove that we should go with #57, I don't know what would :P

**Comment by jeffkaufman** [comment]
@tnelling 
> If that doesn't prove that we should go with #57, I don't know what would

#57 also has conflicts.

This is something that will be fixed by moving away to named bonuses, however we handle them.

**Comment by tnelling** [comment]
Oh right never mind.

**Comment by jeffkaufman** [comment]
Resolved merge conflicts.  Not currently planning on doing the same to #57 since I think #57 is silly, but if someone asks me to I will.

**Comment by jeffkaufman** [comment]
also hold off until #33 is in, because #33 will have conflicts as well

File: players/jeffkaufman/bonuses/thank-tnelling-for-58 [hunk 1]
```diff
@@ -0,0 +1 @@
+-1
```
Comment (sligocki) [inline players/jeffkaufman/bonuses/thank-tnelling-for-58]: What is +-1?

**Comment by jeffkaufman** [comment]
Merged in the deletions from #68 and #69, and updated assuming #67 will merge.

**Pull request merged**